cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,13 @@
+class AdressStepPage {
+    private checkoutButton: string;
+
+    constructor() {
+      this.checkoutButton = "[name = 'processAddress']";
+    }
+  
+    public proceedToCheckoutIII(): void {
+        cy.get(this.checkoutButton).click();
+    }
```

Comment:
There's no need to call as "III"
```suggestion
    public proceedToCheckout(): void {
        cy.get(this.checkoutButton).click();
    }
```